packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { node, number, oneOf, string } from 'prop-types';
+
+import { Description, PopoverContainer, Title, Wrapper } from './styles';
+
+function Popover({
+  children,
+  title,
+  description,
+  position,
+  width,
+  height,
+  ...props
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
```

Comment:
We can simplify here 
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
I agree that we can simplify, but I'm used to setting the variables names as descriptive as possible, so there should be not doubt about what's open when reading the code.

WDYT?

Reply:
For me, it is already clear that we are in a component that controls visibility for something, so the `isOpen` is related to it. 

No problem to go with your solution anyways